packages/fxa-auth-server/lib/senders/emails/partials/button/index.scss

```diff
@@ -0,0 +1,28 @@
+@use '../../global.scss';
+
+@mixin text-lg {
+  font-size: 18px !important;
+  line-height: 28px !important;
+}
+$blue-background: #0a84ff !important;
+
+.p-button {
+  background-color: $blue-background;
+  width: 310px !important;
+  border-radius: 4px !important;
+}
+
+.p-button p {
+  font-family: global.$font-sans;
+  background: $blue-background;
+  @include text-lg;
+}
+
+.p-button td {
+  background: $blue-background;
+}
+
+.p-button a {
+  text-decoration: none;
+  color: #fff;
```

Comment:
Let's go ahead and set a variable to all colors. This one is fine as `$white` per the design guide.